recipes/r-multbxxc/meta.yaml

```diff
@@ -0,0 +1,106 @@
+{% set version = '1.0.1' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-multbxxc
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/multbxxc_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/multbxxc/multbxxc_{{ version }}.tar.gz
+  sha256: f5911f334d3390cbb7747fa6db95a7c792e74117c493acadf15752c12457e5c6
+
+build:
+  merge_build_host: True  # [win]
+  # If this is a new build for the same version, increment the build number.
+  number: 0
+
+  # This is required to make R link correctly on Linux.
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+# Suggests: testthat 
+requirements:
+  build:
+    - {{ compiler('c') }}              # [not win]
+    - {{ compiler('m2w64_c') }}        # [win]
+    - {{ compiler('cxx') }}            # [not win]
+    - {{ compiler('m2w64_cxx') }}      # [win]
+    - {{posix}}filesystem        # [win]
+    - {{posix}}make
+    - {{posix}}sed               # [win]
+    - {{posix}}coreutils         # [win]
+    - {{posix}}zip               # [win]
+
+  host:
+    - r-base
+    - r-rcpp >=1.0.0
+    - r-rcpparmadillo
+    - r-rmumps >=5.2.1_6
+    - libblas
+    - liblapack
+
+  run:
+    - r-base
+    - {{native}}gcc-libs         # [win]
+    - r-rcpp >=1.0.0
+    - r-rcpparmadillo
+    - r-rmumps >=5.2.1_6
+    - libblas
+    - liblapack
+
+test:
+  commands:
+    # You can put additional test commands to be run here.
+    - $R -e "library('multbxxc')"           # [not win]
+    - "\"%R%\" -e \"library('multbxxc')\""  # [win]
+
+  # You can also put a file called run_test.py, run_test.sh, or run_test.bat
+  # in the recipe that will be run at test time.
+
+  # requires:
+    # Put any additional test requirements here.
+
+about:
+  home: https://metasys.insa-toulouse.fr/software/influx/
+  license: GPL (>= 2)
+  summary: 'Contains auxiliary routines for influx software. This packages is not intended to
+    be used directly. Influx was published here: Sokol et al. (2012) <doi:10.1093/bioinformatics/btr716>.'
+
+  license_family: GPL3
```

Comment:
Use `GPL` instead of `GPL3`.

Reply:
Cleaning up is done. Thanks for your ant work.

Reply:
No problem @sgsokol  - thank you for contributing to CF as always :)